csaf_2.0/register/rolie-feed-type/csaf.md

```diff
@@ -0,0 +1,7 @@
+I suggest to register "csaf" as ROLIE feed type per the following record satisfying the requirements given at [RFC 8322, Section 8.4 "ROLIE Information Types Registry"](https://www.rfc-editor.org/rfc/rfc8322.html#section-8.4):
+
+```
+Name: csaf
+Index: <Will be assigned by IANA>
+Reference: https://docs.oasis-open.org/csaf/csaf/v2.0/cs02/csaf-v2.0-cs02.html#7115-requirement-15-rolie-feed
```

Comment:
Please use the link of CSAF as OASIS standard... (at least that was needed last time.)
```suggestion
Reference: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#7115-requirement-15-rolie-feed